source/horoscope/css/landing.css

```diff
@@ -0,0 +1,124 @@
+#landing-title {
+  font-size: 4em;
+  font-weight: bolder;
+  color: white;
+  margin-top: 40px;
+  /* adjust the spacing between the letters */
+  letter-spacing: 0.25em;
+
+}
+
+img {
+    /* send to back */
+    z-index: -1;
+
+    /* take up the full page, independant of other items */
+    position: absolute;
+
+    /* relatively center itself */
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
```

Comment:
not working properly either because of css rule not applying or because the go button is below the screen and thus causes that section to not have background 
![image](https://github.com/StephenTan12/Horoscope-Team-5/assets/59126064/6f07e774-a9df-46f1-b1bf-f69b1288544a)

Reply:
good catch - my previous code set the size of the background equal to size of the window, so if the window was small enough to scroll, this would occur